setup.py

```diff
@@ -46,6 +43,16 @@ def get_description():
 REQUIREMENTS = ['spyder>=3.2.0', 'pexpect', 'tornado',
                 'coloredlogs', 'requests']
 
+
+# Verify that COMPONENTS exist before trying to build the wheel
+if any([arg == 'bdist_wheel' for arg in sys.argv]):
+    if not osp.isdir(COMPONENTS):
+        print("\nWARNING: Server components are missing!! Please run "
+              "'python setup.py sdist' first.\n")
+        sys.exit(1)
+
+
+# Add pywinpty to our Windows dependencies when building wheels
 if os.name == 'nt' or any([arg.startswith('win') for arg in sys.argv]):
     REQUIREMENTS.append('pywinpty>=0.1.3')
```

Comment:
Maybe we should update this to take into account pywinpty v0.2.1

Reply:
Is it really necessary? I mean, did you make changes here that forcefully require using pywinpty 0.2.1? Or do you think is simply better to move people to pywinpty 0.2.1?

Reply:
Everything works well under 0.1.4, however, 0.2 was broken, and those errors were corrected on 0.2.1, so it would be better to update the library on such cases.

Reply:
Ok, I updated it in my last commit.